lib/discord/op/event/reaction_event.py

```diff
@@ -0,0 +1,164 @@
+"""
+reaction_event.py
+------------------------
+リアクションに変化があったときの処理が入っている。
+"""
+
+import discord
+
+from lib.data.tweet_votes_record import TweetsVoteRecord
+from lib.discord.tweet_vote_utils import create_tweet_vote_embed
+from lib.logging.logger import log
+from lib.settings.setting import Setting
+
+
+class ReactionEvent:
+
+    RESPOND_REQUIRED = 0
+    NO_RESPOND_REQUIRED = 1
+    ROLLBACK_REQUIRED = 2
```

Comment:
`IntEnum` にするのはどうでしょう
```suggestion
from enum import IntEnum

class ReactionEventRequirement(IntEnum):
    RESPOND = 0
    NO_RESPOND = 1
    ROLLBACK = 2

class ReactionEvent:
```